src/main/java/org/jenkinsci/plugins/pipeline/github/PullRequestGroovyObject.java

```diff
@@ -501,6 +501,29 @@ public void removeAssignees(final List<String> assignees) {
         }
     }
 
+    @Whitelisted
+    public void review(final String event) {
+        review(null, event, null);
+    }
+
+    @Whitelisted
+    public void review(final String event, final String body) {
+        review(null, event, body);
+    }
+
+    @Whitelisted
+    public void review(final String commitId, final String event, final String body) {
```

Comment:
Do you think you could create a `Map<String, Object>` version of this method? This would allow us to use the groovy idiomatic way of calling this method.

```groovy
pullRequest.review(commitId: 'abc', event: 'REQUEST_CHANGES', body: 'blah blah')
```